docs/DMS/DMS-AUTH.md

```diff
@@ -7,10 +7,10 @@ Also see [Authentication and Authorization Requirements](../AUTH.md)
 
 ## Claimsets
 
-In order to keep the JWT from becoming unreasonably large, and to
-prevent managing detailed permissions within the OAuth 2.0 provider, these
-schemes may use a role and/or "claimset" name that can then be mapped to
-detailed information.
+In order to keep the JWT from becoming unreasonably large, and to prevent
+managing detailed permissions within the OAuth 2.0 provider, these schemes may
+use a role and/or "claimset" name that can then be mapped to detailed
+information.
```

Comment:
Let's try to put the claimset name as a [Scope](https://www.rfc-editor.org/rfc/rfc6749#section-3.3). Even though the client won't be requesting a scope (at least not today's existing clients), we are allowed to put in a scope:

> The authorization server MAY fully or partially ignore the scope requested by the client, based on the authorization server policy or the resource owner's instructions.  If the issued access token scope is different from the one requested by the client, the authorization server MUST include the "scope" response parameter to inform the client of the actual scope granted.